### PR TITLE
Fill DS website with Content

### PR DESCRIPTION
**Pull request by Bassel17**

### CODE REVIEW

File: website/data/pages/accessibility.mdx [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
+
+It’s not completely true for now, but it’s an objective we’re aiming for. It’s important that Strapi product and plugins are usable and useful to everyone. <br/> <br/>
+To be inclusive means we must consider accessibility at the core of how we design and develop. <br/> <br/>
+Products and digital experiences must be accessible to everyone, including those with vision, hearing, cognitive, or mobility impairments.
+
+## Building experiences for everyone
+
+Using our components is a way to improve accessibility and consistency when building products for Strapi.
+
+## Assistive technology support
```
Comment (mfrachet) [inline website/data/pages/accessibility.mdx:20]: That's not true. Even less with the rush of the v4. I would, personally, not put it here

File: website/data/pages/typography.mdx [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+# Typography
+
+Typography is an important part of the Design System that brings consistency across experiences and platforms. Good typography rules help present content clearly and efficiently.
+
+## Typeface
+
+Strapi Design System uses system fonts to provide the best experience on every Operating System (OS). Due to that chance, a performance issue is also reduced. <br/> <br/>
+Moreover, each level of the Type Scale is defined by the Greek alphabet to abstract all context from each level and developed in rem instead of px. Thus, any usecase should be covered by the scale.
+
+![Typeface](/typeface.png)
```
Comment (mfrachet) [inline website/data/pages/typography.mdx:15]: The typographies are not the valid ones

File: website/data/pages/grid-system.mdx [hunk 3]
```diff
@@ -1,3 +1,42 @@
...
+
+![Baseline](/baseline.png)
+
+This baseline convention naturally flows on to the rest of the grid system, which includes icons, components, and layout dimensions. Always try to align objects on the 8px grid, but where necessary use good judgement to fine tune your designs to 4px. The 4px baseline is there to allow more flexibility for line heights and smaller adjustments.
+
+All our “components blocks” flows vertically along this baseline, creating a similar rhythm across all screens. 
+
+## Spacing System
+
+Use multiples of 8px when defining measurements, spacing, and positioning elements.
```
Comment (mfrachet) [inline website/data/pages/grid-system.mdx:22]: Multiples of 4 , not 8

File: website/components/Navbar/index.js [hunk 4]
```diff
@@ -24,17 +24,24 @@ const Navbar = ({ navigationContent }) => {
...
-            DS! Forum
-          </StyledLink>
-        </Box>
       </SubNavSections>
+      <Box
+        paddingLeft={4}
+        paddingBottom={4}
+        style={{
+          position: 'absolute',
+          bottom: 0,
```
Comment (mfrachet) [inline website/components/Navbar/index.js:33]: you can use:

- position="absolute"
- bottom={0} directly

File: website/data/pages/radius-&-shadows.mdx [hunk 5]
```diff
@@ -1,3 +1,34 @@
...
+We use it on 3 differents cases : <br/> <br/>
+**‣**  On the content containers (in addition with the border below) <br/>
+**‣**  On the Filters <br/> 
+**‣**  On the PopOvers & Modals <br/> 
+
+## Table & Content container border
+
+In addition to the related drop shadow, we add a slight border on the main container, PopOvers, Modals, HelpBox and every other elements we want to highlight or make pop over the interface.
+
+![Table & Content container Border](/table-and-content-container.png)
```
Comment (mfrachet) [inline website/data/pages/radius-&-shadows.mdx:34]: We don't have carbon colors

File: website/data/pages/radius-&-shadows.mdx [hunk 6]
```diff
@@ -1,3 +1,34 @@
...
+
+## Drop shadows
+
+In order to keep consistency across the product, the shadows also follow certain rules.
+
+![Drop shadows](/drop-shadows.png)
+We use it on 3 differents cases : <br/> <br/>
+**‣**  On the content containers (in addition with the border below) <br/>
+**‣**  On the Filters <br/> 
+**‣**  On the PopOvers & Modals <br/>
```
Comment (mfrachet) [inline website/data/pages/radius-&-shadows.mdx:28]: Popover not PopOver

**Review by mfrachet — CHANGES_REQUESTED** [review]
Hey, thank you for this 👍🏻 .

I find it a bit of a shame not to have relied on the MDX capabilities (with regular react components) instead of PNGs. This would have improved the experience + making the documentation more interactive and inclusive.

I got a bunch of comments on the side

File: website/pages/[slug].js [hunk 7]
```diff
@@ -10,7 +10,15 @@ import components from 'components/MarkdownComponents';
...
     <Layout meta={frontMatter} navigationContent={navbarContent || {}}>
-      <Box padding={10} background="neutral0" shadow="filterShadow" hasRadius={true}>
+      <Box
+        padding={10}
+        background="neutral0"
+        shadow="filterShadow"
+        hasRadius={true}
+        style={{
+          maxWidth: '880px',
+        }}
```
Comment (mfrachet) [inline website/pages/[slug].js:20]: you can use "maxWidth" directly

File: website/pages/index.js [hunk 8]
```diff
@@ -12,7 +12,15 @@ import PropTypes from 'prop-types';
...
     <Layout meta={{ title: 'Welcome' }} navigationContent={navbarContent || {}}>
-      <Box padding={10} background="neutral0" shadow="filterShadow" hasRadius={true}>
+      <Box
+        padding={10}
+        background="neutral0"
+        shadow="filterShadow"
+        hasRadius={true}
+        style={{
+          maxWidth: '880px',
+        }}
```
Comment (mfrachet) [inline website/pages/index.js:22]: same

**Comment by mfrachet** [comment]
Also, there are multiple h1 / pages which is not very recommended in html

File: website/data/pages/accessibility.mdx (hunk 1 above)
Comment (HichamELBSI) [reply website/data/pages/accessibility.mdx:20]: I agree !

File: website/data/pages/principles.mdx [hunk 9]
```diff
@@ -2,28 +2,27 @@
...
+> A design system is a collection of pre-built, reusable assets like components, patterns that allows its users to build consistent digital experiences faster. By using the pre-built and universal assets of the Design System, the time teams spend designing and building is greatly reduced and users can invest time on high value missions instead of repetitive tasks.
 
-> ## Parts is open-source
+> ## Design System is open-source
 
-> Parts is funded and built by Strapi, which means that the primary reason that we built it is to meet the company’s business needs, but we’ve made it open source for anyone to use and contribute back to. While being primarily open source, Carbon also serves various parts of the IBM business that follow an inner source model.
+> Design System is funded and built by Strapi, which means that the primary reason that we built it is to meet the company’s business needs, but we’ve made it open source for anyone to use and contribute back to. While being primarily open source, Carbon also serves various parts of the IBM business that follow an inner source model.
```
Comment (HichamELBSI) [inline website/data/pages/principles.mdx:19]: `The Design System ...`

File: website/data/pages/principles.mdx [hunk 10]
```diff
@@ -2,28 +2,27 @@
...
+It's an open source design system for plugins, products and digital experiences. This design system consists of design principles and guidelines, created from our design vision, components, design tools and resources, and a vibrant community of contributors.
 
 > ## What is a Design System?
 
-> A design system is a collection of pre-built, reusable assets like components, patterns that allows its users to build consistent digital experiences faster. By using the pre-built and universal assets of Parts!, the time teams spend designing and building is greatly reduced and users can invest time on high value missions instead of repetitive tasks.
+> A design system is a collection of pre-built, reusable assets like components, patterns that allows its users to build consistent digital experiences faster. By using the pre-built and universal assets of the Design System, the time teams spend designing and building is greatly reduced and users can invest time on high value missions instead of repetitive tasks.
 
-> ## Parts is open-source
+> ## Design System is open-source
```
Comment (HichamELBSI) [inline website/data/pages/principles.mdx:17]: `The Design System ...`

**Comment by philippkhon** [comment]
> Hey, thank you for this 👍🏻 .
> 
> I find it a bit of a shame not to have relied on the MDX capabilities (with regular react components) instead of PNGs. This would have improved the experience + making the documentation more interactive and inclusive.
> 
> I got a bunch of comments on the side

Hello @mfrachet, thanks for your comment. Sure, I agree that such workarounds as using simple PNGs instead of proper React components is not a great solution, so we will rework it after the release in a proper way. However, as we were limited in time, we had no other option than that.

Regarding the content - I will double check it again with @maevalienard, thanks.

File: website/data/pages/typography.mdx (hunk 2 above)
Comment (philippkhon) [reply website/data/pages/typography.mdx:15]: @maevalienard could you pls inform me where to find the valid typography? Thanks.

File: website/data/pages/radius-&-shadows.mdx (hunk 6 above)
Comment (philippkhon) [reply website/data/pages/radius-&-shadows.mdx:28]: it's spelled as PopOver in the [UI Kit in Figma](https://www.figma.com/file/PICiE8O4NLrHO1lhJftLdE/Strapi---UI-Kit-%F0%9F%A7%A9?node-id=4183%3A64018). @maevalienard what is the correct variant ?
image

Comment (philippkhon) [reply website/data/pages/radius-&-shadows.mdx:28]: It's spelled as PopOver in the [UI Kit in Figma](https://www.figma.com/file/PICiE8O4NLrHO1lhJftLdE/Strapi---UI-Kit-%F0%9F%A7%A9?node-id=4183%3A64018). @maevalienard what is the correct variant ?
image

File: website/data/pages/accessibility.mdx (hunk 1 above)
Comment (philippkhon) [reply website/data/pages/accessibility.mdx]: Removed it, thanks

File: website/data/pages/typography.mdx (hunk 2 above)
Comment (philippkhon) [reply website/data/pages/typography.mdx]: Replaced it by the actual typographies from the [UI Kit](https://www.figma.com/file/PICiE8O4NLrHO1lhJftLdE/Strapi---UI-Kit-%F0%9F%A7%A9?node-id=4183%3A64017)

File: website/data/pages/grid-system.mdx (hunk 3 above)
Comment (philippkhon) [reply website/data/pages/grid-system.mdx]: Fixed it, thanks

File: website/data/pages/radius-&-shadows.mdx (hunk 5 above)
Comment (philippkhon) [reply website/data/pages/radius-&-shadows.mdx:34]: Replaced it with **Neutral 150**, thanks

File: website/data/pages/grid-system.mdx (hunk 3 above)
Comment (philippkhon) [reply website/data/pages/grid-system.mdx]: Fixed, thanks

File: website/data/pages/radius-&-shadows.mdx (hunk 6 above)
Comment (maevalienard) [reply website/data/pages/radius-&-shadows.mdx:28]: I usually write it PopOvers, but it doesn't matter that much in that context, both works IMO :)

**Review by maevalienard — APPROVED** [review]

**Pull request merged**